src/cryptoadvance/specter/static/styles.css

```diff
@@ -1244,3 +1244,16 @@ input:checked + .slider:before {
     -ms-transform: translateX(26px);
     transform: translateX(26px);
 }
+
+
+
+/* The viewable height is often smaller than the height given by the browser. 
+See: https://css-tricks.com/the-trick-to-viewport-units-on-mobile/
+This style is used for a div at the bottom in the <main> element to create a distance at the bottom of the pages
+*/
+.mobile-distance-element-botom {
+    height: mobileDistanceElementBotomHeight; /* Fallback for browsers that do not support Custom Properties */
```

Comment:
How is this a fallback? What would such a browser do with "mobileDistanceElementBotomHeight"?

Reply:
I removed it.